### DISCUSSION TITLE
how to set application icon?

### DISCUSSION THREAD

**Discussion by Arukenofu** [Deluze/electron-vue-template#54] [Q&A] [answered]
![изображение](https://github.com/Deluze/electron-vue-template/assets/126886598/55f27d72-54ea-4f2b-975e-767bfd1701c0)
![изображение](https://github.com/Deluze/electron-vue-template/assets/126886598/aff86880-bff8-4e6c-8355-2088f1caddb8)

**Comment by Deluze** [comment, marked as answer]
You're setting the icon for the window itself, if you want to give an icon to your application, you'll have to do this in the electron-builder config.

See: https://www.electron.build/configuration/configuration
